cxx/distributions/bigram.hh

```diff
@@ -50,12 +50,14 @@ class Bigram : public Distribution<std::string> {
     for (size_t i = 0; i != indices.size() - 1; ++i) {
       transition_dists[indices[i]].incorporate(indices[i + 1]);
     }
+    ++N;
```

Comment:
Can you add checking N's value to the bigram's test? (I did this in my CL for a few of the distributions).

Reply:
Done.